examples/mistral/mistral_fp16_optimize.json

```diff
@@ -0,0 +1,109 @@
+{
+    "input_model": {
+        "type": "PyTorchModel",
+        "config": {
+            "hf_config": {
+                "model_name": "mistralai/Mistral-7B-v0.1",
+                "model_class": "MistralForCausalLM"
+            }
+        }
+    },
+    "systems": {
```

Comment:
nit: this `systems` and `engine.target` are not really needed since Olive can infer from the `CUDAExecutionProvider` that the system is local gpu.